push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/Sender.java

```diff
@@ -0,0 +1,84 @@
+package org.jboss.aerogear.unifiedpush.message;
+
+import java.util.List;
+
+import javax.enterprise.event.Event;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import org.jboss.aerogear.unifiedpush.api.PushMessageInformation;
+import org.jboss.aerogear.unifiedpush.api.Variant;
+import org.jboss.aerogear.unifiedpush.api.VariantMetricInformation;
+import org.jboss.aerogear.unifiedpush.message.jms.Dequeue;
+import org.jboss.aerogear.unifiedpush.message.jms.DispatchToQueue;
+import org.jboss.aerogear.unifiedpush.message.sender.NotificationSenderCallback;
+import org.jboss.aerogear.unifiedpush.message.sender.PushNotificationSender;
+import org.jboss.aerogear.unifiedpush.message.sender.SenderTypeLiteral;
+import org.jboss.aerogear.unifiedpush.utils.AeroGearLogger;
+
+public class Sender {
```

Comment:
This is a last piece in chain of processors that sends notifications:

````
SenderServiceImpl > TokenLoader > Sender
````

I agree it causes confusion, needs javadoc and better name. Can you think of something @matzew @sebastienblanc  ?